app/controllers/admin/publishers_controller.rb

```diff
@@ -118,7 +118,6 @@ def refresh_uphold
     connection = UpholdConnection.find_by(publisher: params[:publisher_id])
     if connection.present?
       connection.sync_connection!
-      connection.create_uphold_cards
```

Comment:
`sync_connection!` previously conditionally called `create_uphold_cards`.  Since it now calls `create_uphold_cards` whenever there's a successful result, this line is unnecessary.